GNUmakefile

```diff
@@ -59,13 +59,14 @@ chainlink: $(SGX_BUILD_ENCLAVE) operator-ui contracts ## Build the chainlink bin
 	CGO_ENABLED=0 go run packr/main.go "${CURDIR}/core/eth" ## embed contracts in .go file
 	go build $(GOFLAGS) -o $@ ./core/
 
-.PHONY: contracts 
+.PHONY: contracts
 contracts: # build the required evm contracts
 	yarn workspace chainlinkv0.5 setup
 	yarn workspace chainlink setup
 
 .PHONY: operator-ui
 operator-ui: ## Build the static frontend UI.
+	yarn workspace @chainlink/styleguide run build
```

Comment:
@HenryNguyen5 I don't like that this is here, but one of the CI checks fails without this being here. Is it okay if it's left here?

Reply:
Hm, ill take a look at this tomorrow, not sure why this would be failing. Do you know which CI check was failing?

Reply:
It was this commit https://app.circleci.com/github/smartcontractkit/chainlink/pipelines/c4fa7776-f6f0-44eb-baa9-9ef351cd702d/workflows/a7322f0d-fb4b-41d4-83b4-18b3eb5fad53